engine/api/workermodel/registration.go

```diff
@@ -79,10 +78,8 @@ func updateAllToCheckRegistration(db gorp.SqlExecutor) error {
 
 // UpdateSpawnErrorWorkerModel updates worker model error registration
 func UpdateSpawnErrorWorkerModel(db gorp.SqlExecutor, modelID int64, spawnError sdk.SpawnErrorForm) error {
-	// some times when the docker container fails to start, the docker logs is not empty but only contains utf8 null char
-	rs := strings.NewReplacer(string([]byte{0x00}), "", string([]byte{0xbf}), "")
-	spawnError.Error = rs.Replace(spawnError.Error)
-	spawnError.Logs = []byte(rs.Replace(string(spawnError.Logs)))
+	spawnError.Error = sdk.RemoveNotPrintableChar(spawnError.Error)
+	spawnError.Logs = []byte(sdk.RemoveNotPrintableChar(string(spawnError.Logs)))
```

Comment:
`0xbf` seems is not removed by `RemoveNotPrintableChar `

https://play.golang.org/p/FNT_J5qgl7J

Reply:
osef
this is enough to fix the problem pq utf8 conversion problem we had

finally, this has nothing to deal with the current PR. Don't block it

If you want to fix all weird utf8 unicode problem, fix it in venom, this function is coming from it.